bits/missing_certificate/missing_certificate.py

```diff
@@ -1,28 +1,19 @@
-from typing import List, Iterator, Union
+from typing import Iterator
 
 from octopoes.models import OOI
-from octopoes.models.ooi.certificate import Certificate
 from octopoes.models.ooi.findings import KATFindingType, Finding
-from octopoes.models.ooi.service import IPService
 from octopoes.models.ooi.web import Website
 
 
 def run(
-    input_ooi: IPService,
-    additional_oois: List[Union[Certificate, Website]],
+    input_ooi: Website,
+    additional_oois,
 ) -> Iterator[OOI]:
 
-    if input_ooi.service.tokenized.name != "https":
+    if input_ooi.ip_service.tokenized.service.name.lower() != "https":
```

Comment:
The list of ip_services should be larger. imaps, pops etc all also use certs.
However, in the future the ip_service should contain a 'uses ssl/tls' flag which we fingerprint for.

Reply:
That's true indeed. There are much more SANs to be implemented. For now, I've only implemented these few to keep the PR/ issue small. I'll create another ticket to implement more SANs and refer to this PR and issue.